### ISSUE TITLE
Failed to lock apt for exclusive operation

### ISSUE THREAD

**Issue by 030** [suzuki-shunsuke/ansible-git-lfs-ubuntu#3]
```
TASK [suzuki-shunsuke.git-ubuntu : Install git] *****************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failed to lock apt for exclusive operation"}
```

**Comment by 030** [comment]
Could you explain the purpose of:

```
---
# vars file for git-lfs-ubuntu
git_lfs_nonroot: "{{ (ansible_env.HOME == '/root') | ternary('no', 'yes') }}"
```

**Comment by 030** [comment]
It seems that there is a dependency on git. I have already installed git using another role. Could the dependency be removed?

**Comment by suzuki-shunsuke** [comment]
@030 

Thank you for your feedback!
And sorry for the late reply.

This repository wasn't maintained for a long time and its quality wasn't good.
So I have rewritten this role and released the new version v1.1.0.

I have removed the variable `git_lfs_nonroot` and the dependency on `git`.

Thank you.

**Comment by suzuki-shunsuke** [comment]
I close this issue.
If you have any question about this issue, please reopen it.
